code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -33,8 +33,8 @@
 
 /obj/item/projectile/plasma/adv
 	damage = 30
-	range = 37
-	mine_range = 37
+	range = 15
+	mine_range = 15
```

Comment:
Hey there, can you nerf the mining range back to standard levels? I feel like the plasma cutter should be a hand cannon rather than a powerful god weapon.
I feel like the bullets should go to the edge of the screen and just that. So the buff doesn't buff everything about it too much.

Reply:
That fine

Reply:
Fixed